docs/internals/contributing.rst

```diff
@@ -51,7 +51,7 @@ For Windows the use of cmd under Administrator permission is suggested instead o
 
 .. code-block:: batch
 
-    > virtualenv -p "C:\Python34\python.exe" env
+    > C:\Python34\python.exe -m venv env
```

Comment:
A full path is currently already assumed to be that location. So while still not fully accurate, I do think this is an improvement on what is currently in the tutorial.